example/use_cases/employer_university_flow/pkg/issuer.go

```diff
@@ -0,0 +1,76 @@
+package pkg
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/example"
+	"github.com/sirupsen/logrus"
+)
+
+func BuildExampleUniversityVC(universityID string, recipient string) (*credential.VerifiableCredential, error) {
```

Comment:
Can we remove this and make the one below public?

Reply:
done.